spec/classes/dhcp_spec.rb

```diff
@@ -22,6 +22,7 @@
         osfamily: 'RedHat',
         operatingsystem: 'RedHat',
         operatingsystemrelease: '6',
+        operatingsystemmajrelease: '6',
```

Comment:
Could you add a test for when the major release is 7, adding in the detection of the service file? 👍